SECURITY.md

```diff
@@ -1,3 +1,90 @@
+Security issues happen as part of the normal lifecycle of software development of OpenSearch. This document describes the process for reporting these issues, responding to these issues, and ensuring we treat them not only with the appropriate priority but also with respect for the discoverers, software providers and their users, and the OpenSearch community in general.
+
+If you discover a potential security issue in this project we ask that you notify the OpenSearch Security Team via email to opensearch-security@amazon.com. Please do **not** create a public GitHub issue.
+
+*Giving credit where credit is due, this policy is heavily influenced by the [Xen Project’s security response process](https://xenproject.org/developers/security-policy/), that was put to the test during the [embargo period for XSA-108 back in 2014](https://xenproject.org/2014/10/22/xen-project-security-policy-improvements-get-involved/) and improved its clarity around managing the pre-disclosure list and the deployment of fixes during embargo. We are standing on the shoulders of these battle-tested giants.*
+
+## The Security Team
+
+The OpenSearch Security Team is a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. Maintainers can be added to the Security Team by submitting a PR updating this document and adding their name to the list below (the process for becoming a maintaner can be found [here](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer)).
+
+The OpenSearch Security Team will address reported issues on a best effort basis, prioritizing them based on several factors, including severity.
+
+### Current Members
+
+| Security Team member     | GitHub Alias                                  | Affiliation |
+| ------------------------ | --------------------------------------------- | ----------- |
+| Dave Lago                | [davidlago](https://github.com/davidlago)     | Amazon      |
+| Charlotte Henkle         | [CEHENKLE](https://github.com/CEHENKLE)       | Amazon      |
+| Daniel (dB.) Doubrovkine | [dblock](https://github.com/dblock)           | Amazon      |
+| Anirudha (Ani) Jadhav    | [anirudha](https://github.com/anirudha)       | Amazon      |
+| Sean Neumann             | [seanneumann](https://github.com/seanneumann) | Amazon      |
+| Anan Zhuang              | [ananzh](https://github.com/ananzh)           | Amazon      |
+| Eli Fisher               | [elfisher](https://github.com/elfisher)       | Amazon      |
+
+## Process
+
+Anyone finding an issue that is already publicly disclosed (for example, a CVE in one of the project’s dependencies) should feel free to create an issue and discuss openly on GitHub. The process below is only intended for issues that have not been publicly disclosed yet.
+
+1. We request that instead of opening a GitHub issue, it is reported via email at opensearch-security@amazon.com.
```

Comment:
would it make sense to list here the necessary information which should be provided (if available)? namely a reproduction case?

Reply:
Yeah... good point. Let me add to that section with some high level items to think about/include when reporting an issue.